tools/signing/sign.targets

```diff
@@ -0,0 +1,61 @@
+<Project DefaultTargets="Sign">
+  <Import Project="..\dependencies.props" />
+  <PropertyGroup>
+    <NuGetPackageRoot>$(RestorePackagesPath)</NuGetPackageRoot>
+    <NuGetPackageRoot Condition="!HasTrailingSlash('$(NuGetPackageRoot)')">$(NuGetPackageRoot)\</NuGetPackageRoot>
+    <SignType Condition="'$(SignType)' == ''">test</SignType> 
+
+    <_DryRun>true</_DryRun>
+    <_DryRun Condition="'$(BUILD_BUILDNUMBER)' != ''">false</_DryRun>
+    <_TestSign>false</_TestSign>
+    <_TestSign Condition="'$(SignType)' == 'test'">true</_TestSign>
```

Comment:
Maybe you can change this to just use some logic around SignType

Reply:
Could you elaborate? I thought that is what I was doing.

Reply:
I meant to remove the `_TestSign` property and just use `SignType` instead. But looking again I think that might just make things more complicated anyway..